src/throttle/exceptions.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from django.core.exceptions import PermissionDenied, ImproperlyConfigured
+
+class RateLimitExceeded(PermissionDenied):
+    pass
+
+class ThrottleCoolDownNotDefined(ImproperlyConfigured):
+    pass
+
+class ThrottlePreventFunctionNotDefined(ImproperlyConfigured):

```

Comment:
this is actually so wrong, for a while we don't use `PreventFunction` since the function itself is related with user